api/dbadapters/bigquery.ts

```diff
@@ -167,7 +177,11 @@ export class BigQueryDbAdapter implements IDbAdapter {
           : metadata.type === "VIEW"
           ? dataform.TableMetadata.Type.VIEW
           : dataform.TableMetadata.Type.UNKNOWN,
-      target,
+      target: {
+        database: metadata.tableReference.projectId,
+        schema: metadata.tableReference.datasetId,
+        name: metadata.tableReference.tableId
+      },
```

Comment:
Could be worth adding a throw in if the input target and this derived one don't match, otherwise we could end up with a feedback loop of consistently referencing the wrong table when `table` is called.